crates/bevy_render/src/view/mod.rs

```diff
@@ -91,8 +89,6 @@ pub struct ViewUniform {
     inverse_view: Mat4,
     projection: Mat4,
     world_position: Vec3,
-    near: f32,
-    far: f32,
```

Comment:
This is a type used in shader bindings. Specifically it maps to the View struct in various shaders. If you remove it here, it must be removed there too else this will break rendering. I did a quick check in pbr.wgsl and it doesn't seem like the view near/far are used anywhere in there, but please do a thorough check before removing it from the shaders.

Reply:
Thank you for noticing. I removed them from every View type I found and checked every .wgsl, .vert and .frag file and none of them used either the near or far values.